### PR TITLE
Fixed inconsistent bolding of text

### PR DESCRIPTION
**Pull request by drpayyne**
## Purpose of this pull request

This pull request (PR) fixes the inconsistent bolding of text

## Affected DevDocs pages
- https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/css-topics/css-overview.html
- https://devdocs.magento.com/guides/v2.2/frontend-dev-guide/css-topics/css-overview.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: guides/v2.2/frontend-dev-guide/css-topics/css-overview.md [hunk 1]
```diff
@@ -32,7 +32,7 @@ There are a couple options to help with CSS and site performance.
 
 * Minification of CSS files reduces the file size being sent. It does this by stripping white space within the file.
 
-To enable / disable these settings, go into Admin > **Stores** > Setting > **Configuration** > **Advanced** > **Developer** > **CSS Settings**.
+To enable / disable these settings, go into **Admin** > **Stores** > **Settings** > **Configuration** > **Advanced** > **Developer** > **CSS Settings**.
```
Comment (hguthrie) [inline guides/v2.2/frontend-dev-guide/css-topics/css-overview.md]: Since those two words that you bolded are technically not something a user would click, I suggest cleaning this entire sentence:

```suggestion
You can enable or disable these settings in the _Admin_ panel.  Click **Stores** and, in the _ Settings_ section, click **Configuration** > **Advanced** > **Developer** > **CSS Settings**.
```

Comment (drpayyne) [reply guides/v2.2/frontend-dev-guide/css-topics/css-overview.md]: Telling the developer "and in the _Settings_ section", seems too wordy, and can rather be communicated in this better way, perhaps?
To enable or disable these settings, go to _Admin_ panel, click **Stores** > **Settings::Configuration** > **Advanced** > **Developer** > **CSS Settings**.

@rogyar Any input on this from your side? Thanks.

Comment (hguthrie) [reply guides/v2.2/frontend-dev-guide/css-topics/css-overview.md]: Sure! But definitely remove that forward slash. 🙂

File: guides/v2.2/frontend-dev-guide/css-topics/css-overview.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ In the Blank theme, the buttons of the `.action.primary` class, so called *prima
 OrangeCo wants to change the color of the primary buttons to orange. To achieve this, they do the following:
 
 1. Create a new Orange theme, which inherits from the Blank [theme](https://glossary.magento.com/theme).
-2. In the Orange theme directory add the overriding `app/design/frontend/OrangeCo/orange/web/css/source/_theme.less` file with the following code:
+1. In the Orange theme directory add the overriding `app/design/frontend/OrangeCo/orange/web/css/source/_theme.less` file with the following code:
```
Comment (drpayyne) [inline guides/v2.2/frontend-dev-guide/css-topics/css-overview.md]: @dobooth Why was the 2nd point made `1` as well? Is this a typo? It's not present in my commit.

Comment (hguthrie) [reply guides/v2.2/frontend-dev-guide/css-topics/css-overview.md]: In markdown, ordered lists need only the number 1. In fact, if in the future we need to insert a line, we don't have to renumber ever other line after. Therefore, all of our ordered list lines begin with 1.

Comment (drpayyne) [reply guides/v2.2/frontend-dev-guide/css-topics/css-overview.md]: I understand @hguthrie, but what's the need to replace the following
`1. Sample`
`2. Sample`
**with**
`1. Sample`
`1. Sample` ?
